src/main/java/UI/UI.java

```diff
@@ -0,0 +1,68 @@
+package UI;
+
+import Task.TaskList;
+
+import java.util.Scanner;
+
+/**
+ * Handles user interactions such as displaying messages and receiving input.
+ * Hello!
+ */
+public class UI {
+    private final Scanner input;
+
+    /**
+     * Constructs a UI object for handling user input.
+     */
+    public UI() {
+        input = new Scanner(System.in);
+    }
+
+    /**
+     * Displays a welcome message to the user.
+     */
+    public void showWelcomeMessage() {
+        System.out.println("Hello! I'm Bob\nWhat can I do for you?");
+        System.out.println();
+    }
+
+    /**
+     * Displays a message indicating a task has been added and prints the list of tasks.
+     */
+    static void taskAddedMsg() {
+        System.out.println("Sure! I've added that in for you.");
+        TaskList.mainTaskList.printList();
+    }
+
+    /**
+     * Displays a separator line to the console.
+     */
+    public void showSeparator() {
+        System.out.println("--------------------------------------------------");
```

Comment:
Can we try setting this as a final field?
```suggestion
        private static final DIVIDER = "--------------------------------------------------"
        // Code
        ...
        System.out.println(DIVIDER);
```